installer/scripts/service_control

```diff
@@ -59,6 +108,33 @@ setup_variables()
     OMSADMIN_CONF=$CONF_DIR/omsadmin.conf
 
     OMSAGENT_WS=omsagent-$WORKSPACE_ID
+
+    WS_STATUS=0
+}
+
+validate_omsadmin_conf() {
```

Comment:
Please use the method definition style with a newline before the opening bracket:
```
method()
{
...
}
```

Reply:
I think I got all those in this source file.  See next push.